app/code/Magento/Backend/Test/Mftf/ActionGroup/AssertAdminPersistentShoppingCartOptionsAvailableActionGroup.xml

```diff
@@ -9,8 +9,6 @@
 <actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
               xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
     <actionGroup name="AssertAdminPersistentShoppingCartOptionsAvailableActionGroup">
-
-        <amOnPage url="{{PersistentShoppingCartConfig.url}}" stepKey="navigateToPersistencePage"/>
         <conditionalClick selector="{{AdminPersistentShoppingCartSection.DefaultLayoutsTab}}" dependentSelector="{{AdminPersistentShoppingCartSection.CheckIfTabExpand}}" visible="true" stepKey="clickTab"/>
```

Comment:
You can also move that line to AdminNavigateToPersistentShoppingCartSettingsActionGroup, as a result of that action group we have configuration grid ready to work

Reply:
Done